.github/workflows/task-unit-test.yml

```diff
@@ -17,6 +17,10 @@ on:
         description: 'Run valgrind tests'
         type: boolean
         default: true
+      run-codecov:
```

Comment:
is this temporary?

Reply:
it defaults to false and might be useful in the future